lib/matplotlib/table.py

```diff
@@ -331,8 +328,7 @@ def _get_grid_bbox(self, renderer):
 
         Only include those in the range (0,0) to (maxRow, maxCol)"""
         boxes = [self._cells[pos].get_window_extent(renderer)
```

Comment:
Since the value is actually used here, it might make sense to use `items()` below.